submissions/kovalov/html-css-popup/index.html

```diff
@@ -0,0 +1,211 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1.0"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="header__container">
+        <nav class="navigation">
+          <ul class="navigation__list">
+            <li class="navigation__list-item">
+              <a href="#" class="navigation__list-link"> Mail </a>
+            </li>
+            <li class="navigation__list-item">
+              <a href="#" class="navigation__list-link"> Images </a>
+            </li>
+          </ul>
+        </nav>
+        <div class="dropdown">
+          <input
+            class="dropdown__control"
+            type="checkbox"
+            id="dropdown-control-1"
+            tabindex="-1"
```

Comment:
This attribute broke the native browser order of interactive elements, use it wisely.